feat: provide PageProps, LayoutProps types (#13308)
sveltejs/kit: documentation/docs/20-core-concepts/20-load.md

```

> [!LEGACY]

> > Before version 2.16.0, the props of a page/layout have to be typed individually:

Reviewer: ```suggestion
> Before version 2.16.0, the props of a page and layout had to be typed individually:
```

</aside>
```

> [!LEGACY]

> > `LayoutProps` was added in 2.16.0, in earlier versions props can be typed like so:

Reviewer: ```suggestion
> `LayoutProps` was added in 2.16.0. In earlier versions, props had to be typed individually:
```
